src/dmd/target.d

```diff
@@ -60,6 +60,7 @@ struct Target
         bool cppExceptions;       /// set if catching C++ exceptions is supported
         char int64Mangle;         /// mangling character for C++ int64_t
         char uint64Mangle;        /// mangling character for C++ uint64_t
+        bool deletingDestructor;  /// target ABI uses deleting destructor
```

Comment:
Haha, I just remembered why I didn't call it `cppDeletingDestructor` (which was the original name), it's because adding 3 characters caused the commend to move to the right... and then there was the terrible tension; do I move ALL the comments to the right to maintain the line-up, or do I let my comment be the only one out of line? ;)

Reply:
I opted for the conservative resolution of; use a shorter variable name...